tests/src/Hodor/FlowTest.php

```diff
@@ -44,7 +44,13 @@ public function testJobCanBeRan()
         $bin_path = __DIR__ . '/../../../bin';
 
         $job_name = 'job-name-' . uniqid();
-        $job_params = ['the_time' => date('c'), 'known_value' => 'donuts'];
+        $job_params = [
+            'the_time'    => date('c'),
+            'known_value' => 'donuts',
+            'job_options' => [
+                'run_after' => date('c'),
```

Comment:
Hi Sir,

That actually needs to be a string because it gets `json_encode`d and passed to the `test-publisher` script as a string. The script then `json_decode`s it and creates a `DateTime` out of it. 

The integration test is doing so much that's not particularly clear. Better tests are needed, but right now the FlowTest was a minimal amount of work that covers the main functional parts of Hodor.

Thanks for the review and question :smile: 

Lightster